Cabal/Distribution/Simple/Configure.hs

```diff
@@ -413,20 +445,28 @@ configure (pkg_descr0', pbi) cfg = do
                 allConstraints
                 (dependencySatisfiable
                     (fromFlagOrDefault False (configExactConfiguration cfg))
+                    (packageVersion pkg_descr0)
                     installedPackageSet
                     internalPackageSet
                     requiredDepsMap)
                 comp
                 compPlatform
                 pkg_descr0
 
+    debug verbosity $ "Finalized package description:\n"
+                  ++ showPackageDescription pkg_descr
```

Comment:
It's debug mode, and it really isn't that much text (compared to some of the other stuff we spew, even in verbose non-debug mode!) And I've only ever found this information useful.